docs/user-guide/deploying/custom-deployment.md

```diff
@@ -42,7 +42,7 @@ Most of the changes made in the manager_config.json will not be visible to the d
 :::
 
 ### Asset type (/model)
-Create your own asset type that fits your product. In the asset type you define its name, icon, and color, and set its attributes with configuration items (called meta items in the code). If you need some inspiration, you can look at OpenRemote's [default asset types](https://github.com/openremote/openremote/tree/master/model/src/main/java/org/openremote/model/asset/impl).
+Create your own asset type that fits your product. In the asset type you define its name, icon, and colour, and set its attributes with configuration items (called meta items in the code). If you need some inspiration, you can look at OpenRemote's [default asset types](https://github.com/openremote/openremote/tree/master/model/src/main/java/org/openremote/model/asset/impl).
```

Comment:
It's the preferred spelling in US English. :wink:

Currently both UK and US English spelling is mixed up in the docs/code which makes it a bit messy IMHO. Similarly you'll find both initlialize/initialise used.

Reply:
Yes, I usually write in US English but here I saw colours used above, so I aligned (at least on this page).
We should define if we want to write US or UK English and stick to it everywhere.